strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxPersistence.java

```diff
@@ -86,6 +86,18 @@ public synchronized EntityManagerFactory createContainerEntityManagerFactory(Per
         return new OJPAPartitionedEntityManagerPool(properties);
     }
 
+    @Override
+    public void generateSchema(PersistenceUnitInfo info, Map map)
+    {
+
```

Comment:
Same as above.

I believe we should `throw new UnsupportedOperationException` for each new method from the persistence API already not supported by us.